python/sglang/srt/server_args.py

```diff
@@ -974,6 +975,11 @@ def add_cli_args(parser: argparse.ArgumentParser):
             action="store_true",
             help="Disable Multi-head Latent Attention (MLA) for DeepSeek V2/V3/R1 series models.",
         )
+        parser.add_argument(
+            "--disable-multimodal",
+            action="store_true",
+            help="Disable Multi-head Latent Attention (MLA) for DeepSeek V2/V3/R1 series models.",
```

Comment:
It should has nothing to do with DeepSeek. Is it right?

Reply:
I copy-pasted it from the above line 😂. The help info is fixed.